server/validateReviews.js

```diff
@@ -0,0 +1,23 @@
+import { body,validationResult } from "express-validator";
+export default function validateReviews(req,res){
```

Comment:
This function is getting a couple of concepts mixed up, and ends up not really doing anything, before finally returning `undefined` (nothing).